### PR TITLE
Upgrade llvm in warning as errors CI tests

### PR DESCRIPTION
**Pull request by t-b**

### CODE REVIEW

File: cppapi/server/fwdattribute.cpp [hunk 1]
```diff
@@ -133,10 +133,10 @@ void FwdAttribute::set_att_config(const Tango::AttributeConfig_5 &conf)
 // Enum labels (in case of)
 //
 
-    enum_labels.clear();
```
Comment (bourtemb) [inline cppapi/server/fwdattribute.cpp]: This new indentation looks strange on github page showing the differences (this page).
It was not aligned before, but it still does not seems to be aligned with the code above (line 130) after.

File: cppapi/server/attrgetsetprop.cpp [hunk 2]
```diff
@@ -440,8 +440,8 @@ void Attribute::add_config_5_specific(AttributeConfig_5 &conf)
 
 void Attribute::set_properties(const Tango::AttributeConfig &conf,std::string &dev_name,TANGO_UNUSED(bool from_ds),std::vector<AttPropDb> &v_db)
 {
-    if (name_lower == "state" || name_lower == "status")
-        return;
+  if (name_lower == "state" || name_lower == "status")
+    return;
```
Comment (bourtemb) [inline cppapi/server/attrgetsetprop.cpp]: 2 spaces indentation instead of 4?

File: cppapi/server/fwdattribute.cpp (hunk 1 above)
Comment (bourtemb) [reply cppapi/server/fwdattribute.cpp]: I guess this is again due to the mixing between tabs and spaces.

**Comment by bourtemb** [comment]
> Looks good! Could you please re-start failed jobs? 

I restarted them. I had to log out from Travis and log in again to be able to see the restart job buttons. Strange.

> Also, ubuntu 20.04 build started to fail, but this is unrelated to the changes you made in this PR.

Indeed. We have a an issue with 20.04 which is trying to install tzdata and his asking some questions during the install/configuration process.

**Comment by t-b** [comment]
@bourtemb I'll look into that. How wide is a tab at cppTango? 4 spaces?

**Comment by bourtemb** [comment]
> @bourtemb I'll look into that. How wide is a tab at cppTango? 4 spaces?

I would say 4 but I'm pretty sure it's not consistent all over the place. And in some files tabs are used instead of spaces.
The recommendation up to now was to fit the style of the surrounding code.
In this particular case, it looks like the surrounding code is using tabs, so tabs should be used here I would say?

Hopefully, when https://github.com/tango-controls/cppTango/issues/586 will be addressed, all these indentation issues will disappear.

**Comment by t-b** [comment]
@bourtemb Thx.

**Comment by t-b** [comment]
@mliszcz @bourtemb Updated.

**Comment by t-b** [comment]
The important travis task passed, see https://travis-ci.org/github/tango-controls/cppTango/jobs/678571433.

**Pull request merged**